File: pubnub-java/src/integrationTest/java/com/pubnub/api/integration/pam/PamServerIntegrationTest.java
package com.pubnub.api.integration.pam;

public class PamServerIntegrationTest extends AccessManagerIntegrationTest {

    @Override
    public String getPamLevel() {
        return LEVEL_APP;
    }

    @Override
    boolean performOnServer() {
        return true;
    }
}

File: src/integrationTest/kotlin/com/pubnub/api/integration/pam/PamServerIntegrationTest.kt
package com.pubnub.api.integration.pam

import com.pubnub.api.PubNub

class PamServerIntegrationTest : AccessManagerIntegrationTest() {

    override fun getPamLevel() = LEVEL_APP

    override fun performOnServer() = true

    override val pubnubToTest: PubNub = server
}
